dataAccess.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+// Fetch data from the members.json file
+const loadMembers = () => {
+    const jsonData = fs.readFileSync(__dirname + "/" + "members.json");
+        return JSON.parse(jsonData);
+}
+
+// Update members.js file
```

Comment:
These comments doesn't add additional context, please delete them
```suggestion

const loadMembers = () => {
    const jsonData = fs.readFileSync(__dirname + "/" + "members.json");
        return JSON.parse(jsonData);
}

```

Reply:
Done!